tests/snapshots/run_file__wrong_string.hvm.snap

```diff
@@ -0,0 +1,9 @@
+---
+source: tests/golden_tests.rs
+input_file: tests/golden_tests/run_file/wrong_string.hvm
+---
+Invalid readback:
+Invalid String Character value '(*, 4)'
+Invalid String Character value '*'
+
+""
```

Comment:
Shouldn't the result still be readback as `(SCons (*, 4) (SCons * SNil))`?

Oh, this is happening on master too. I thought #146 had fixed this. Should be fixed later then